tracer/build/_build/Build.Steps.cs

```diff
@@ -602,6 +603,18 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
                 var workingDirectory = ArtifactsDirectory / $"linux-{LinuxArchitectureIdentifier}";
                 EnsureCleanDirectory(workingDirectory);
 
+                if (!IsArm64)
+                {
+                    var tracerNativeFile = MonitoringHomeDirectory / "Datadog.Trace.ClrProfiler.Native.so";
+                    var newTracerNativeFile = MonitoringHomeDirectory / "tracer" / "Datadog.Tracer.Native.so";
```

Comment:
The profiler native binary is called: `Datadog.AutoInstrumentation.Profiler.Native.{architecture}.dll`
The tracer native binary is called: `Datadog.Trace.ClrProfiler.Native.so`

I feel like we should unify those names before we ship this, one way or another. I'm not passionate either way, but I think we should pick one

My _only_ concern about the `*AutoInstrumentation*` section, is the name starts to get long, and we don't want to be bitten by `MAX_PATH` 😅